post_installation_configuration/coreos-layering.adoc

```diff
@@ -0,0 +1,111 @@
+
+[IMPORTANT]
+====
+Some Hotfixes require a Red Hat Support Exception and are outside of the normal scope of {product-title} support coverage or life cycle policies.
+====
+
+In the event you need a Hotfix, you will be provided with the Hotfix package, based on link:https://access.redhat.com/solutions/2996001[Red Hat Hotfix policy]. Apply it on top of the base image and test that new custom layered image in a non-production environment. When you are satisfied that the custom layered image is safe to use in production, you can roll it out on your own schedule to selected node pool. For any reason, you can easily roll back the custom layered image and return to using the default {op-system}.    node
+
+[NOTE]
+====
+It is planned for future releases that you can use {op-system} layering to incorporate third-party software packages such as libreswan or numactl.
+====
+
+////
+Future features
+By using layering, you can extend your {op-system} in a number of ways, including:
+
+* {op-system} Hotfixes
+* Third-party RHEL packages.
+* Bleeding edge drivers and kernel enhancements to improve performance or add capabilities.
+* Foresic client tools to investigate possible and actual break-ins.
+* Inventory agents that provide a coherent view of the entire fleet.
+* Critical and important CVEs as soon as errata are available in RHEL to keep your systems secure.
+* SSH Key management packages.
+////
+
+To apply a custom layered image, you create a Containerfile that references a {product-title} image and the Hotfix that you want to apply. For example:
+
+// For example, the following Containerfile installs a Hotfix:
+
+.Example Containerfile to apply a Hotfix
+[source,yaml]
+----
+# Using a 4.12.0 image
+FROM quay.io/openshift-release-dev/ocp-release@sha256:6499bc69a0707fcad481c3cb73225b867d
+#Install hotfix rpm
+RUN rpm-ostree override replace https://example.com/myrepo/haproxy-1.0.16-5.el8.src.rpm && \
+    rpm-ostree cleanup -m && \
+    ostree container commit
+----
+
+////
+For example, the following Containerfile installs RHEL's libreswan package:
+[source,terminal]
+----
+# Using aa 4.12.0 image
+FROM quay.io/openshift-release-dev/ocp-release@sha256:6499bc69a0707fcad481c3cb73225b867dc31b345c6e6204e28
+RUN rpm-ostree install libreswan && \
+    rpm-ostree cleanup -m && \
+    ostree container commit
+----
+////
+
+[NOTE]
+====
+You should use the same base {op-system} image that in installed on the rest of your cluster. Use the `oc adm release info --image-for rhel-coreos-8` command to obtain the base image being used in your cluster.
```

Comment:
Maybe it should be "image that **is** installed" instead of "image that **in** installed"